app/controllers/tag_controller.rb

```diff
@@ -436,4 +438,14 @@ def location_modal
   def gridsEmbed
     render layout: false
   end
+
+  private
```

Comment:
I would just do `private :order_string` at the end of the `order_string` method.  That will make `order_string` private.  The way you have it now, anything method below this line will be private.  Let's just make `order_string` private so no future developer needs to worry about it. :)

Reply:
But all other controllers in this project have used same pattern to specify private methods in a class.
You can check out the home_controller.rb, application_controller.rb files, etc. So i think its better to do it the same way. : smile